specification.scs

```diff
@@ -0,0 +1,18 @@
+knowledge_base_ims_specification
+    <- concept_reusable_component_specification;;
+
+knowledge_base_ims_specification = [*
+knowledge_base_ims
+    <- concept_reusable_component;
+    <- concept_nonatomic_reusable_component;
+    <- concept_independent_reusable_component;
+    <- concept_reusable_kb_component;
+    <- concept_reusable_dynamically_installed_component;
+    <- concept_reusable_source_code_component;
+
+    => nrel_sc_identifier: [IMS knowledge base](* <- lang_en;; *);
+    => nrel_component_dependencies: ... (* <- empty_set ;; *);
+
+    => nrel_component_address: [https://github.com/SHtress/ims.ostis.kb] (* <- github_url;; *);
```

Comment:
ostis-ai, not stress
```suggestion
    => nrel_component_address: [https://github.com/ostis-ai/ims.ostis.kb] (* <- github_url;; *);
```

Reply:
Forget to update it after testing, thanks for reminding